GSoC-Ideas.md

```diff
@@ -1,5 +1,65 @@
 # Ideas for Google Summer of Code projects
 
+## Idea #x: Implementing CHAOSS metrics with Perceval
+
+[ Micro-tasks and place for questions ](#)
+
+### Description
+
+The [GMD Working group](https://github.com/chaoss/wg-gmd) is proposing some metrics
+that are computed with information obtained from software development repositories.
+One of the goals of the working group is to provide reference implementations of
+those metrics, based on the output produced by [Perceval](https://github.com/chaoss/grimoirelab-perceval).
+As an example, still work in progress, check the Python notebook with the
+[reference implementation for commits](https://github.com/chaoss/wg-gmd/blob/master/implementations/Code_Commits.ipynb).
+
+The aims of this idea are as follows:
+
+* Producing Python (Jupyter) notebooks with proposals for reference implementations
+for these metrics. The proposals will follow the usual acceptance mechanism of the working group
+(based on GitHub issues and pull requests). In general, all reference implementations will
+build on the output produced by Perceval for the relevant data source,
+and will explore the peculiarities of the metric and its implementation for that data source.
+
+* Document the notebooks as much as possible, so that any person trying to implement the metric
+can understand not only how to implement it, but also the details that should have into account.
```

Comment:
line 25: This sentence I don't understand; it seems incomplete: "but also the details that should have into account."

Reply:
My bad. Is it more clear now?